code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -46,6 +46,7 @@
 	..()
 
 //Green
+/*
```

Comment:
If you're going to remove slime mutation toxin, set the rabidness of slimes back to their default, or don't remove them but edit the mutations so they don't have some of the cheese associated with them

Reply:
no

Reply:
Because?

Reply:
bluespace tiles, aka faster tiles i just rename them to something else